client/web-dev-server.config.mjs

```diff
@@ -38,7 +38,8 @@ export default {
       include: ['src/utils/config.js'],
       preventAssignment: false,
       __api_url__: (process.env.API_URL || 'http://localhost:8000') + '/api',
-      __purchase_mode__: process.env.AVOCANO_PURCHASE_MODE,
+      __purchase_mode__: process.env.AVOCANO_PURCHASE_MODE || 'default',
```

Comment:
This, specifically the default value of `'default'` will need to be updated in `rollup.config.js` as well. It's currently going to make this value empty (Will update my earlier comment)

Reply:
Fixed in [d33b2f8](https://github.com/GoogleCloudPlatform/avocano/pull/155/commits/d33b2f8d360a7bcbfcee8f59493f355b863a7ad3)